src/components/CategoryPicker/index.js

```diff
@@ -1,47 +1,91 @@
-import React, {useMemo} from 'react';
-import _ from 'underscore';
+import React, {useMemo, useState} from 'react';
 import {withOnyx} from 'react-native-onyx';
+import _ from 'underscore';
+import lodashGet from 'lodash/get';
 import ONYXKEYS from '../../ONYXKEYS';
 import {propTypes, defaultProps} from './categoryPickerPropTypes';
-import OptionsList from '../OptionsList';
 import styles from '../../styles/styles';
-import ScreenWrapper from '../ScreenWrapper';
 import Navigation from '../../libs/Navigation/Navigation';
 import ROUTES from '../../ROUTES';
+import CONST from '../../CONST';
+import * as IOU from '../../libs/actions/IOU';
+import * as OptionsListUtils from '../../libs/OptionsListUtils';
+import OptionsSelector from '../OptionsSelector';
+import useLocalize from '../../hooks/useLocalize';
+
+function CategoryPicker({policyCategories, reportID, iouType, iou, policyRecentlyUsedCategories}) {
+    const {translate} = useLocalize();
+    const [searchValue, setSearchValue] = useState('');
+
+    const policyCategoriesCount = _.size(policyCategories);
+    const isCategoriesCountBelowThreshold = policyCategoriesCount < CONST.CATEGORY_LIST_THRESHOLD;
```

Comment:
I think it makes more sense.

```suggestion
    const isCategoriesCountLessThreshold = policyCategoriesCount < CONST.CATEGORY_LIST_THRESHOLD;
```

Reply:
Hum, I think that's worse in terms of english correctness 🤔

Reply:
Haha, okay, I am not an English guru. Let's keep it 😅